lib/symmetry.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:wqhub/wq/wq.dart' as wq;
+
+enum Symmetry {
+  identity,
+  rotate1,
+  rotate2,
+  rotate3,
+  mirror1,
+  mirror2,
+  diagonal1,
+  diagonal2;
+
+  // Utilities
+
+  static wq.Point transformPoint(wq.Point p, Symmetry symmetry, int boardSize) {
```

Comment:
Sorry, I should have been more specific. I meant to make it a member function, so that the `symmetry` parameter is not necessary.

Additionally, it would be nice to unify code related to handling symmetries and affine transforms. For example, currently the task search feature also uses a similar approach, so we could remove it from there and use `Symmetry` as well (see https://github.com/ale64bit/WeiqiHub/blob/main/lib/train/task_repository.dart#L339-L345).